README.md

```diff
@@ -1,20 +1,48 @@
 # Jaeger Kubernetes Templates
 
-Support for deploying Jaeger into Kubernetes.
+## Development setup
+This template uses an in-memory storage with a limited functionality for local testing and development.
+Do not use this template in production environments.
 
-## All-in-one template
+Install everything in the current namespace:
+```bash
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-kubernetes/master/all-in-one/jaeger-all-in-one-template.yml
+```
+
+Once everything is ready, `minikube service jaeger-query --url` tells you where to find Jaeger URL.
+
+## Production setup
+This template deploys all Jaeger components as a separate services: StatefulSet Cassandra storage, agent as DaemonSet,
+collector and query service with UI as Deployments. Each one of those can be managed and scaled individually.
+
+```bash
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-kubernetes/master/production/jaeger-production-template.yml
+```
+
+Note that it's OK to have the Query and Collector pods to be in an error state for the first minute or so. This is
+because these components attempt to connect to Cassandra right away and hard fail if they can't after N attempts.
+
+Your Agent hostname is `jaeger-agent.${NAMESPACE}.svc.cluster.local`
+
+Once everything is ready, `minikube service jaeger-query --url` tells you where to find Jaeger URL.
```

Comment:
The same as above. Do not make assumption that users are using minikube (and this is prod template).